src/Arduboy.cpp

```diff
@@ -26,7 +27,32 @@ void Arduboy::start() // deprecated
 void Arduboy::begin()
 {
   boot(); // required
+  bootUtils();
 
+  bootLogo();
+
+  // Audio
+  tunes.initChannel(PIN_SPEAKER_1);
```

Comment:
The dev kit pins didn't change in the code.  Are you compiling with DEVKIT and not PROD?